### PR TITLE
Enhancement for Hue Init action test

### PR DESCRIPTION
**Pull request by prince-cs**
Adding the hue test to submit the pyspark job with a python script which verifies, the basic hue functionalities.

### CODE REVIEW

File: hue/run_queries.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+execute_and_wait(db, dbms,
+                 "INSERT INTO TABLE store VALUES ('a', 1), ('b', 2), ('c', 3)")
+query_handle = execute_and_wait(db, dbms,
+                                'SELECT food FROM store WHERE price < 3')
+result = db.fetch(query_handle)
+for row in result.rows():
+    # pylint: disable=superfluous-parens
+    print(row)
+
+exit()
```
Comment (functicons) [inline hue/run_queries.py:28]: This is not necessary.

File: hue/verify_hue_running.py [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+                'username': username,
+                'password': password,
+                'csrfmiddlewaretoken': self._session.cookies['csrftoken'],
+                'next': next_url
+            },
+            cookies={},
+            headers={'Referer': full_path})
+        response.raise_for_status()
+
+    def CheckMetadata(self, wait_seconds=5, max_retries=20):
```
Comment (functicons) [inline hue/verify_hue_running.py:71]: make naming conventions consistent, use check_metadata here. Update other places as well.

**Review by functicons — CHANGES_REQUESTED** [review]

File: hue/run_queries.py (hunk 1 above)
Comment (prince-cs) [reply hue/run_queries.py]: done

File: hue/verify_hue_running.py (hunk 2 above)
Comment (prince-cs) [reply hue/verify_hue_running.py]: done

**Review by functicons — APPROVED** [review]

**Comment by functicons** [comment]
Presbumit failed:

```
2024-01-22T18:01:04.967134280Z FAILED (errors=1)
2024-01-22T18:01:04.967157156Z ================================================================================
2024-01-22T18:01:04.967164892Z ==================== Test output for //hue:test_hue (shard 1 of 3):
2024-01-22T18:01:04.967172159Z Running tests under Python 3.8.10: /usr/bin/python3
2024-01-22T18:01:04.967179463Z [  FAILED  ] setUpClass (__main__.HueTestCase)
2024-01-22T18:01:04.967186988Z ======================================================================
2024-01-22T18:01:04.967194356Z ERROR: setUpClass (__main__.HueTestCase)
2024-01-22T18:01:04.967201802Z ----------------------------------------------------------------------
2024-01-22T18:01:04.967209684Z Traceback (most recent call last):
2024-01-22T18:01:04.967217285Z   File "/home/ia-tests/.cache/bazel/_bazel_ia-tests/83b1ae36bb04ea5432b9efccee83c25f/execroot/_main/bazel-out/k8-fastbuild/bin/hue/test_hue.runfiles/_main/integration_tests/dataproc_test_case.py", line 62, in setUpClass
2024-01-22T18:01:04.967239800Z     assert cls.PROJECT
2024-01-22T18:01:04.967247834Z AssertionError
```

**Pull request merged**